color-guessing game/java/main.js

```diff
@@ -1,105 +0,0 @@
-var allboxes= document.querySelectorAll('.box');
-var x=document.querySelector('#rgbspan');
-var color=generalRandomColor(6);
-var selectedColor=color[Math.floor(Math.random()*6)];

```

Comment:
whats this @sona-mathew-08

Reply:
I didn't add that one. I don't know how it got added. But I had deleted the main.js file.

Reply:
@sona-mathew-08 actually its the js file of another project please restore it and push the changes 
https://github.com/arpit456jain/Amazing-Js-Projects/blob/master/color-guessing%20game/java/main.js